test/Microsoft.OpenApi.Readers.Tests/V2Tests/OpenApiParameterTests.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers.ParseNodes;
+using Microsoft.OpenApi.Readers.V2;
+using SharpYaml.Serialization;
+using Xunit;
+
+namespace Microsoft.OpenApi.Readers.Tests.V2Tests
+{
+    [Collection("DefaultSettings")]
+    public class OpenApiParameterTests
+    {
+        private const string SampleFolderPath = "V2Tests/Samples/OpenApiParameter";
+
+        [Fact]
+        public void ParseBodyParameterShouldSucceed()
+        {
+            using (var stream = File.OpenRead(Path.Combine(SampleFolderPath, "bodyParameter.yaml")))
+            {
+                var yamlStream = new YamlStream();
+                yamlStream.Load(new StreamReader(stream));
+                var yamlNode = yamlStream.Documents.First().RootNode;
```

Comment:
A lot of codes are duplicated. Think to add a private method to wrapper?

Reply:
I'll look into this now.